standard/expressions.md

```diff
@@ -1033,6 +1081,8 @@ Given an implicit conversion `C₁` that converts from an expression `E` to a ty
 - `E` exactly matches both or neither of `T₁` and `T₂`, and `T₁` is a better conversion target than `T₂` ([§12.6.4.6](expressions.md#12646-better-conversion-target))
 - `E` is a method group ([§12.2](expressions.md#122-expression-classifications)), `T₁` is compatible ([§20.4](delegates.md#204-delegate-compatibility)) with the single best method from the method group for conversion `C₁`, and `T₂` is not compatible with the single best method from the method group for conversion `C₂`
 
+The  conversion from an implicitly-typed out variable declaration is not considered better than any other conversion from expression.
```

Comment:
While I know what this means, it doesn't feel terribly clear - maybe we just need a reference to the "conversion from expression" section?

Reply:
I'm also not sure it adds anything. Should this paragraph be removed? For discussion.